src/list-grid.scss

```diff
@@ -0,0 +1,234 @@
+@import './new-settings';
+@import './mixins';
+
+/*!
+.fd-grid-list
+*/
+$block: #{$fd-namespace}-grid-list;
+
+.#{$block} {
+  $item: #{$block}__item;
+  $more: #{$block}__more;
+
+  $fd-grid-list-border-radius: var(--sapElement_BorderCornerRadius);
+  $fd-grid-list-active-text-color: var(--sapList_Active_TextColor);
+  $fd-grid-list-item-box-shadow: var(--sapContent_Shadow0);
+  $fd-grid-list-mode-box-shadow: 0 0 0 0.125rem var(--sapList_SelectionBorderColor);
+  $fd-grid-list-item-selected-background-color: var(--sapList_SelectionBackgroundColor);
+  $fd-grid-list-item-active-background-color: var(--sapList_Active_Background);
+  $fd-grid-list-item-hover-background-color: var(--sapList_Hover_Background);
+  $fd-grid-list-highlight: (
+    'success': ('backgroundColor': var(--sapSuccessBorderColor)),
+    'error': ('backgroundColor': var(--sapErrorBorderColor)),
+    'warning': ('backgroundColor': var(--sapWarningBorderColor)),
+    'neutral': ('backgroundColor': var(--sapNeutralBorderColor))
+  );
+
+  @include fd-reset();
+  @include fd-flex(column);
+
+  &__item {
+    @include fd-flex(column);
+    @include fd-fiori-focus();
+
+    background-color: var(--sapList_Background);
+    border-radius: $fd-grid-list-border-radius;
+    box-shadow: $fd-grid-list-item-box-shadow;
+    height: 100%;
+    position: relative;
+
+    &--body {
+      @include fd-flex(column);
+
+      height: 100%;
+      justify-content: space-between;
+
+      &--padding {
+        padding: 1rem;
+      }
+    }
+
+    &--counter {
+      color: var(--sapContent_MarkerTextColor);
+    }
+
+    .#{$block}__checkbox--label,
```

Comment:
![image](https://user-images.githubusercontent.com/26483208/98676185-91cc1800-235b-11eb-9556-174d9449b16a.png)
Focus in checkbox inside grid list looks weird

Reply:
fixed